eve/commands/cmd_cal.py

```diff
@@ -0,0 +1,28 @@
+import click
+
+from eve.service import svc_cal
+
+
+class Context:
+    def __init__(self):
+        self.calendar = svc_cal.Calendar()
+
+
+@click.group()
+@click.pass_context
+def cli(ctx):
+    """Google calendar interaction"""
+    ctx.obj = Context()
+
+
+@cli.command()
+@click.pass_context
+def events(ctx):
+    """Display number of upcoming events in google calendar"""
+    result = ctx.obj.calendar.next()
+    for e in result:
+        click.echo(f'{e["name"]}'.center(45, "="))
```

Comment:
I have a preference to add a space between the text and the `=`, like this, `click.echo(f' {e["name"]} '.center(45, "="))`

Reply:
Sure, will do.